### DISCUSSION TITLE
`Relation` field `DISPLAY FIELDS` option

### DISCUSSION THREAD

**Discussion by babakfp** [pocketbase/pocketbase#1765] [Q&A]
Hey

Inside the `posts` collection I'm showing the filename of the video:

![Screenshot 2023-02-03 164716](https://user-images.githubusercontent.com/44144724/216736026-b8432247-afd4-4419-bddc-fa8647f9ef32.png)

Inside the `videos` collection I can't show the filename of the video:

![Screenshot 2023-02-03 164739](https://user-images.githubusercontent.com/44144724/216736031-b38d21af-f55f-4549-870d-5a68e624c886.png)

It is great that I can click on the thumb and watch the video, but the problem is I don't know what video is what unless I play the video. I think it would be better if we could show both (thumb + title) on both the `posts` and the `videos` page.

**Comment by ganigeorgiev** [comment]
Showing the filename alongside the file was requested previously but for now is a very low priority.

I'm not sure about showing the preview in the relation view, since you can technically select more than 1 display fields and I'm not sure how this will look like. Currently we try to "serialize" all display fields as comma separated list of strings.
